angular/src/app/account/send/address/send-address.component.ts

```diff
@@ -43,14 +43,28 @@ export class AccountSendAddressComponent implements OnInit, OnDestroy {
     private fb: UntypedFormBuilder,
     private snackBar: MatSnackBar,
     public translate: TranslateService
-  ) {
-    this.form = fb.group({
+  ) {}
+
+  ngOnDestroy() {}
+
+  async ngOnInit() {
+    // Constructor of send.component.ts should have called resetFee() by now, which sets
+    // the fee to network definition. Here we will attempt to get it from blockchain API.
+    const networkStatus = this.networkStatusService.get(this.sendService.network.id);
+
+    // Grab the fee rate either from network definition or from blockchain API status:
+    if (networkStatus.length > 0) {
```

Comment:
Ah I see what you meant about duplicate the code, I suppose for now this is fine